integration/test_urls.py

```diff
@@ -0,0 +1,70 @@
+"""Integration tests for all urls in Reflex."""
+import os
+import re
+
+import pytest
+import requests
+
+
+def check_urls(repo_dir):
+    """Check that all URLs in the repo are valid and secure.
+
+    Args:
+        repo_dir: The directory of the repo.
+
+    Returns:
+        A list of errors.
+    """
+    url_pattern = re.compile(r'http[s]?://reflex\.dev[^\s"]*')
+    errors = []
+
+    for root, _dirs, files in os.walk(repo_dir):
+        if "__pycache__" in root:
+            continue
+
+        for file_name in files:
+            if not file_name.endswith(".py"):
```

Comment:
should we check markdown files too?

Reply:
great spot, added these in and actually found a few more incorrect paths because of it